modules/statistics/php/stats_behavioural.class.inc

```diff
@@ -245,7 +245,13 @@ class Stats_Behavioural extends \NDB_Form
             $center = $row['CenterID'];
             $vl     = $row['VLabel'];
             $value  = intval($row['val']);
-            $c      =& $this->tpl_data['behaviour']['C' . $center];
+            if (!isset($this->tpl_data['behaviour']['C' . $center])) {
```

Comment:
@driusan the fixes in here maybe should be moved to a separate PR? Solves error_pollution and might not be the prettiest code. @laemtl you wouldn't happen to know a better way? I believe you were the one that cleaned up this file.

Reply:
What about 
$c =& $this->tpl_data['behaviour']['C' . $center][$vl] ?? [$vl => []];

Reply:
@laemtl my IDE is giving me the warning: `Only variables can be passed by reference` message. Can you test that for me whenever you have time and check your error_log? The error log gets polluted when visiting the statistics module and clicking on the Behavioural Statistics tab.

Reply:
@maltheism This shoul works:
$this->tpl_data['behaviour']['C' . $center][$vl] = $this->tpl_data['behaviour']['C' . $center][$vl] ?? [];
$c =& $this->tpl_data['behaviour']['C' . $center];